### DISCUSSION TITLE
HTTP/2 Rapid Reset attack

### DISCUSSION THREAD

**Discussion by balait4** [Kong/kong#11736] [Help] [answered]
Hi All,

Regarding HTTP/2 Rapid Reset attack, This attack exploits a weakness in the HTTP/2 protocol to generate enormous, hyper-volumetric Distributed Denial of Service (DDoS) attacks.

All providers who have HTTP/2 services should assess their exposure to this issue. Software patches and updates for common web servers and programming languages may be available to apply now or in the near future. We recommend applying those fixes as soon as possible.

Similar to below nginx do we have anything for kong already there or anything planned?
 https://www.nginx.com/blog/http-2-rapid-reset-attack-impacting-f5-nginx-products/

**Comment by mheap** [comment, marked as answer]
We’re actively working on patches. Please see https://github.com/Kong/kong/discussions/11741 for more information

**Comment by balait4** [comment]
Thanks
